manifests/g/Glarysoft/FileRecoveryFree/1.17.0.17/Glarysoft.FileRecoveryFree.locale.en-US.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: Glarysoft.FileRecoveryFree
+PackageVersion: 1.17.0.17
+PackageLocale: en-US
+Publisher: Glarysoft Ltd.

```

```suggestion
Publisher: Glarysoft Ltd
```
This must match Control Panel.